### PR TITLE
Add ValueMarkerWidth property

### PR DESCRIPTION
**Pull request by jesusalvino**
### Purpose
Dealing the bug https://jira.autodesk.com/browse/DYN-4825

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [x] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Reviewers
@QilongTang 

### FYIs
@RobertGlobant20 @filipeotero

### CODE REVIEW

**Comment by jesusalvino** [comment]
nodeX

**Comment by mjkkirschner** [comment]
adding another binding for every output to fix this UI bug seems like overkill I guess considering we have performance issues due to excessive binding / events.

File: src/DynamoCoreWpf/ViewModels/Core/OutPortViewModel.cs [hunk 1]
```diff
@@ -130,6 +131,16 @@ public bool PortDefaultValueMarkerVisible
             }
         }
 
+        public string ValueMarkerWidth
```
Comment (mjkkirschner) [inline src/DynamoCoreWpf/ViewModels/Core/OutPortViewModel.cs:134]: /// summary tags for public properties please.

**Comment by mjkkirschner** [comment]
test?

**Comment by QilongTang** [comment]
> adding another binding for every output to fix this UI bug seems like overkill considering we have performance issues due to excessive binding / events.

Agreed, maybe using a converter?

File: src/DynamoCoreWpf/ViewModels/Core/OutPortViewModel.cs (hunk 1 above)
Comment (jesusalvino) [reply src/DynamoCoreWpf/ViewModels/Core/OutPortViewModel.cs]: property removed

**Comment by jesusalvino** [comment]
> test?

Adding

**Review by QilongTang — COMMENTED** [review]
LGTM, let's see if we can add test

**Review by reddyashish — APPROVED** [review]

**Comment by jesusalvino** [comment]
> LGTM, let's see if we can add test

Test done.

**Comment by jesusalvino** [comment]
> test?

Done.

**Comment by jesusalvino** [comment]
imagen

**Review by QilongTang — APPROVED** [review]

**Comment by QilongTang** [comment]
Tests passing at https://master-15.jenkins.autodesk.com/view/DYN/job/DYN-DevCI_Self_Service/1078/

**Pull request merged**